mmtk/Cargo.toml

```diff
@@ -22,7 +22,7 @@ log = {version = "0.4", features = ["max_level_trace", "release_max_level_off"]
 # - change branch/rev
 # - change repo name
 # But other changes including adding/removing whitespaces in commented lines may break the CI.
-mmtk = { git = "ssh://git@github.com/mmtk/mmtk-core.git", rev = "dd332801274240c52ec88bfa0aa8ace5c1d7bc76" }
+mmtk = { git = "ssh://git@github.com/mmtk/mmtk-core.git", rev = "fe8e92c27bf091314106a432efd5b27ac10c40d5" }
```

Comment:
Does this rev change after updates to PR/merge?

Reply:
Yeah. I will fix the rev before merging this PR.